cmd/main.go

```diff
@@ -85,6 +85,8 @@ var (
 	// Serve live profiles of the process (CPU, memory, etc.) over HTTP on this port
 	httpProfilePort   = flag.Int("httpprof_port", 0, "Port to serve HTTP profiles from")
 	foldRemoteRootSvg = flag.Bool("fold_remote_root_svg", false, "Whether to fold root SVG from remote mixer")
+	// Cache SV -> inputPropertyExpressions for StatisticalCalculations
```

Comment:
a bit more detail in the comment on "->"?

Reply:
Done